### PR TITLE
html

### LINKED ISSUE #8: Plain HTML Website
Create a unstyled version of your website. All of the content, links, images, pages, etc. should be correct/working. This will look a lot like the website that you showed me before that you already made.

If you don't know where to start, [khan academy](https://www.khanacademy.org/computing/computer-programming/html-css) seems to have some solid material on this.

Also maybe skim [this html lab](https://github.com/495-Labs-Projects/TempConverter/blob/master/docs/Lab1.pdf) so you know what it contains. There is a chance you will also find [the follow on lab](https://github.com/495-Labs-Projects/TempConverter/blob/master/docs/Lab2.pdf) helpful.

And [w3schools](https://www.w3schools.com/) is often helpful for specific questions.

Name the file for the home page (possibly the index page) `index.html` and **do not put it in a folder**. Work on an appropriate branch and create a pr to close this issue.

### PR DESCRIPTION
**Pull request by hvk5388**

### CODE REVIEW

File: website_outline.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<html>
+
+    <head>
+        <title>hannah kern</title>
+    </head>
+
```
Comment (mjkern) [inline website_outline.html]: You probably want your name here, right?

File: website_outline.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<html>
+
+    <head>
+        <title>hannah kern</title>
```
Comment (mjkern) [inline website_outline.html]: You sure you want this lowercase?

File: website_outline.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    </h4>
+
+    <h5>
+        resume
+    </h5>
+
+    <p>
+        Github icons made by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com</a>
+        Gmail icons made by <a href="https://www.flaticon.com/authors/pixel-perfect" title="Pixel perfect">Pixel perfect</a> from <a href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com</a>
+        LinkedIn icons made by <a href="https://www.flaticon.com/authors/pixel-perfect" title="Pixel perfect">Pixel perfect</a> from <a href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com</a>
```
Comment (mjkern) [inline website_outline.html]: They usually specify exactly what text you must include to use an icon - is this what they said? Do you need the creative commons lisence?

File: website_outline.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    <p>
+        info about clinique
+    </p>
+
+    <h4>
+        Let's Chat!
+    </h4>
+
+    <h5>
+        resume
```
Comment (mjkern) [inline website_outline.html]: This should link to your resume, right?

**Review by mjkern — CHANGES_REQUESTED** [review]
Okay, a few things still:
- this file should be named `index.html` - did you read the issue?
- Can you include the images that you plan to use? I would add them to images folder and then link them via relative paths.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+        hello.
+    </h1>
+
+    <h2>
+        my name is hannah. i am passionate about helping people through innovative design. 
+    </h2>
+
+    <img src = "https://photos.google.com/photo/AF1QipNyZ9mTBYE373cA3-qUuMqOGml62suxp6QN_NCJ">
+
+    <img src = "https://photos.google.com/photo/AF1QipNYwca5fK7wrKjU4tJbM6ZT5NnxRMlUzUSdTZFV">
```
Comment (mjkern) [inline index.html]: These images don't work for me... Also, you should include them in your repo and use a relative path. I would not depend on google for these.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+    <h5>
+        <a href="https://drive.google.com/drive/folders/1dVBwvjCaRg2rCOkvR--Aw_qKzHj-sFX5" resume>
+        </a>
+    </h5>
+
+    <p>
+        Github icons made by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com</a>
+        Gmail icons made by <a href="https://www.flaticon.com/authors/pixel-perfect" title="Pixel perfect">Pixel perfect</a> from <a href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com</a>
+        LinkedIn icons made by <a href="https://www.flaticon.com/authors/pixel-perfect" title="Pixel perfect">Pixel perfect</a> from <a href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com</a>
```
Comment (mjkern) [inline index.html]: I don't see the icons that these refer to...

File: index.html [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+        I worked on creating high fiedlity wireframes for the application.
+    </p>
+
+    <h4>
+        Let's Chat!
+    </h4>
+
+    <h5>
+        <a href="https://drive.google.com/drive/folders/1dVBwvjCaRg2rCOkvR--Aw_qKzHj-sFX5" resume>
+        </a>
```
Comment (mjkern) [inline index.html]: This does not work. I think you should double check the syntax.

File: website_outline.html (hunk 3 above)
Comment (mjkern) [reply website_outline.html]: @hvk5388 You never addressed this. Please at least respond to the comment...

File: website_outline.html [hunk 8]
```diff
@@ -0,0 +1,69 @@
+<html>
```
Comment (mjkern) [inline website_outline.html]: This file should be deleted altogether now that you have one named `index.html`.

**Review by mjkern — CHANGES_REQUESTED** [review]
Better. Still a few notes below

File: index.html [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+        It allows the user to scan their face, view the problem areas, and recommends products and solutions. 
+        I worked on creating high fiedlity wireframes for the application.
+    </p>
+
+    <h4>
+        Let's Chat!
+    </h4>
+
+    <h5>
+        <a href="https://drive.google.com/file/d/1VJn_JMpBiWub_GVBn46kDP1N0Syk42X8/view?usp=sharing">resume</a>
```
Comment (mjkern) [inline index.html]: I don't have permission to follow this link

File: index.html [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+    </h5>
+
+    <img src = "logotype.svg">
+    
+    <img src = "social-media.svg">
+
+    <img src = "email (1).svg">
```
Comment (mjkern) [inline index.html]: This file should be named better

Comment (mjkern) [reply index.html]: actually, all three icons probably should.

And please place all of your images or icons in an folder labeled `images` or `assests` or something.

**Review by mjkern — CHANGES_REQUESTED** [review]
Nice work Hannah! Only a few minor problems

**Review by mjkern — APPROVED** [review]
looks good @hvk5388 

I'm a little surprised that you don't want your resume to open in another tab, but if this is operating how you like it then go ahead and merge :)

**Comment by mjkern** [comment]
@hvk5388 There are no changes since my last review. Did you push?

Also, I am noticing that you never addessed my comments about attribution... can you please do that?

**Comment by hvk5388** [comment]
The attribution is correct

**Comment by mjkern** [comment]
Okay, looks like I just mised your changes before. Sorry about that. And glad the attribution is right :)

**Pull request merged**